index.bs

```diff
@@ -189,7 +189,7 @@ define ways to uniquely identify each one.
 
     <pre highlight="js">
     var sensor = new DirectTirePressureSensor({ position: "rear", side: "left" });
-    sensor.onreading = _ => console.log(sensor.pressure);
```

Comment:
@rwaldron thanks for looking. I will add new example that will use EventTarget interface for onreading, onactivate and onerror. Should I add it to **[Event handlers](https://w3c.github.io/sensors/#event-handlers)** section? What do you think?